Server/src/controllers/getBasicDataController.ts

```diff
@@ -0,0 +1,50 @@
+import { Request, Response } from 'express';
+import { retrieveBasicData } from '../services/getBasicDataService';
+
+export class getBasicDataController {
+    private retrieveBasicDataObject = new retrieveBasicData();

```

Comment:
I think the declaration is fine here but the initialization can happen either in the constructor or in the create method after a request is verified. Since theres not really any verification here constructor can work

Reply:
This is a good catch and an error on my part. It should definitely be in the constructor as that is the convention followed elsewhere in the back end, fixed.
.